woof/transactions.py

```diff
@@ -137,10 +137,10 @@ def _get_topic_from_vertical(vertical):
     return "_".join(["TRANSACTIONS", vertical])
 
 
-def make_kafka_safe(input):
-    if type(input) != unicode:
-        input = str(input)
-        input = input.decode('utf-8')
-        return input.encode('ascii', 'ignore')
+def make_kafka_safe(raw_data):
```

Comment:
Below code can be written this way.

def make_kafka_safe(raw_data):
    return   raw_data.encode('ascii', 'ignore')